cms/migrations/0028_delete_program_courses.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-05-08 19:06
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def remove_program_courses(apps, schema_editor):
+    ProgramCourse = apps.get_model('cms', 'ProgramCourse')
+    ProgramCourse.objects.all().delete()
```

Comment:
I'm not sure we need to delete these ahead of time because the added `course` column is nullable. I think you could just delete them at the start of 0030 instead and save a migration.